doc/onboarding_guide.rst

```diff
@@ -0,0 +1,38 @@
+Onboarding Best Practices
+=========================
+
+You'll have a dev contact on the CLI team. Early and frequent communication with this contact is essential to ensuring a smooth onboarding.
+
+Suggested timeline:
+
+- **First Week:** Reach out to your contact on the CLI team. Set up a short 30 minute Skype call to discuss initial questions.
+
+- **Second Week and as needed:** Create a few commands. Schedule a short 30 minute Skype screen-share to demo your commands. This is crucial! Teams have created entire modules using anti-patterns, resulting in comment-filled PRs and last-minute rework! A quick command review would have prevented this.
+
+- **During command authoring:** Run *check_style --module {your module} --pep8 --pylint* frequently to ensure you are staying on type of style issues that will stall your build when you submit a PR.
```

Comment:
Today, you don't need to append both `--pep8` and `--pylint` anymore. Both will be run by default.

Reply:
I always used --ci which I thought did both, but then it does it on all modules.  Good to know!